src/main/java/MessageConvertor.java

```diff
@@ -1,8 +1,11 @@
 public class MessageConvertor {
-    public JsonMessage parsXmlToJson(XmlMessage a){
-        return new JsonMessage(a.getBody());
+    public JsonMessage parsToJson(MessageType message) {
+        return new JsonMessage(message.getBody());
     }
-    public XmlMessage parsJsonToXml(JsonMessage a){
-        return new XmlMessage(a.getBody());
+
+    public XmlMessage parsToXml(MessageType message) {
+        return new XmlMessage(message.getBody());
     }
```

Comment:
as your are using as input parameter **MessageType** you can collapse you methods to a method named **convert()** that will convert from one type to another